cmd/release/main.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2019 Storj Labs, Inc.
```

Comment:
i think i'd still like to replace this file with the bash script from the other PR

Reply:
Oh yeah. I wanted lay an outline for a future smaller PR.
Should i remove them for now entirely?

Reply:
yeah